includes/internal/emails/class-email-settings-tab.php

```diff
@@ -25,31 +25,49 @@ class Email_Settings_Tab {
 	 * @internal
 	 */
 	public function init() {
-		add_filter( 'sensei_settings_tab_content', [ $this, 'tab_content' ], 10, 2 );
+		add_action( 'sensei_settings_after_links', [ $this, 'render_tabs' ] );
+		add_filter( 'sensei_settings_content', [ $this, 'get_content' ] );
 	}
 
 	/**
-	 * Render the email settings tab.
+	 * Render tabs on the Emails settings page.
 	 *
 	 * @internal
 	 * @access private
 	 *
-	 * @param string $content  Existing content.
 	 * @param string $tab_name The current tab name.
-	 * @return string
 	 */
-	public function tab_content( string $content, string $tab_name ): string {
+	public function render_tabs( string $tab_name ) {
 		if ( 'email-notification-settings' !== $tab_name ) {
-			return $content;
+			return;
 		}
 
 		$current_subtab = $this->get_current_subtab();
 
 		ob_start();
-
 		$this->render_submenu( $current_subtab );
-		$this->render_subtab( $current_subtab );
+		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Output escaped in render_submenu function.
+		echo ob_get_clean();
+	}
 
+	/**
+	 * Get the content on the Emails settings page.
+	 *
+	 * @internal
+	 * @access private
+	 *
+	 * @param string $tab_name The current tab name.
+	 * @return string
+	 */
+	public function get_content( string $tab_name ): string {
```

Comment:
A question came to my mind. What if we have a few "handlers" to this hook? 
We might want to return the initial content if the tab name is not interesting for current handler.
But with recent changes to the hook we can't return that "original" value as we have only the tab name.

Reply:
Fixed in https://github.com/Automattic/sensei/pull/6527/commits/215a11cad0cc74134a52602f7a0edef9081c2801.